selfdrive/car/car_specific.py

```diff
@@ -215,8 +215,8 @@ def create_common_events(self, CS: structs.CarState, CS_prev: car.CarState, extr
       # Enable OP long on falling edge of enable buttons (defaults to accelCruise and decelCruise, overridable per-port)
       if not self.CP.pcmCruise and (b.type in enable_buttons and not b.pressed):
         events.add(EventName.buttonEnable)
-      # Disable on rising and falling edge of cancel for both stock and OP long
-      if b.type == ButtonType.cancel:
+      # Disable on rising edge of cancel for both stock and OP long
+      if b.type == ButtonType.cancel and b.pressed:
```

Comment:
This will fault the Bolt which requires us to not enable on falling edge of cancel button (possible when holding cancel and pressing enable iirc)